src/keys/utils.ts

```diff
@@ -30,7 +30,9 @@ import {
 import * as bip39 from 'bip39';
 import { IdInternal } from '@matrixai/id';
 import * as keysErrors from './errors';
+// TODO: weird problem here
 import { utils as nodesUtils } from '../nodes';
+// import * as nodesUtils from '../nodes/utils';
```

Comment:
This causes a weird error involving `GRPCClientAgent`.

```
    TypeError: Class extends value undefined is not a constructor or null

      20 | interface GRPCClientAgent extends CreateDestroy {}
      21 | @CreateDestroy()
    > 22 | class GRPCClientAgent extends GRPCClient<AgentServiceClient> {
         |                               ^
      23 |   /**
      24 |    * Creates GRPCClientAgent
      25 |    * This connects to the agent service

      at Object.<anonymous> (src/agent/GRPCClientAgent.ts:22:31)
      at Object.<anonymous> (src/nodes/NodeConnection.ts:16:1)
```

Details in: https://github.com/MatrixAI/js-polykey/pull/310#issuecomment-1034784547

Reply:
@tegefaulkes this seems to coming from `tests/grpc/utils`, since the other parts are all using selective imports.

Toggle these lines:

```ts
// TODO: weird problem here
// import { utils as nodesUtils } from '../nodes';
import * as nodesUtils from '../nodes/utils';
```

In `src/keys/utils.ts` and run `npm test -- ./tests/grpc/utils.test.ts` to see if it is passing.

It appears selective imports can affect import ordering, and these side-effects are quite sneaky so only by mass-changing a bunch of selective imports in a particular domain can help.

Reply:
Switching the grpc domain to selective imports seemed to fix this... this time.